skrifa/src/outline/glyf/hint/instance.rs

```diff
@@ -171,13 +171,40 @@ impl HintInstance {
             Definition::default(),
         );
         self.cvt.clear();
+        if let Some(cvar) = outlines.cvar.as_ref() {
+            // First accumulate all the deltas in 16.16
+            self.cvt.resize(outlines.cvt.len(), 0);
+            if let Ok(var_data) = cvar.variation_data(axis_count) {
+                for (tuple, scalar) in var_data.active_tuples_at(coords) {
+                    for delta in tuple.deltas() {
+                        let ix = delta.position as usize;
+                        if let Some(value) = self.cvt.get_mut(ix) {
+                            *value += delta.apply_scalar(scalar).to_bits();
+                        }
+                    }
+                }
+            }
```

Comment:
it's easy to imagine a little helper method that handles accumulating the deltas?

Reply:
good call. I'll add this to the cvar table